component.yaml

```diff
@@ -8,11 +8,10 @@ inputs:
   isRequired: false
```

Comment:
IMO Component name must be "Tuist Install"

Because we have another Tuist component now

Reply:
Yes, "Tuist Install" will be better for this component.